src/integration/car/ev-database/EVDatabaseCarIntegration.ts

```diff
@@ -39,54 +40,96 @@ export default class EVDatabaseCarIntegration extends CarIntegration {
       const chargeStandardTables: CarCatalogConverter[] = [];
       const chargeAlternativeTables: CarCatalogConverter[] = [];
       const chargeOptionTables: CarCatalogConverter[] = [];
-      for (const chargeStandard of Object.keys(data.Charge_Standard_Table)) {
-        const chargeStandardTable: CarCatalogConverter = {
-          type: chargeStandard,
-          evsePhaseVolt: data.Charge_Standard_Table[chargeStandard].EVSE_PhaseVolt,
-          evsePhaseAmp: data.Charge_Standard_Table[chargeStandard].EVSE_PhaseAmp,
-          evsePhase: data.Charge_Standard_Table[chargeStandard].EVSE_Phase,
-          evsePhaseVoltCalculated: data.Charge_Standard_Table[chargeStandard].EVSE_Phase === 3 ? Voltage.VOLTAGE_400 : data.Charge_Standard_Table[chargeStandard].EVSE_PhaseVolt,
-          chargePhaseVolt: data.Charge_Standard_Table[chargeStandard].Charge_PhaseVolt,
-          chargePhaseAmp: data.Charge_Standard_Table[chargeStandard].Charge_PhaseAmp,
-          chargePhase: data.Charge_Standard_Table[chargeStandard].Charge_Phase,
-          chargePower: data.Charge_Standard_Table[chargeStandard].Charge_Power,
-          chargeTime: data.Charge_Standard_Table[chargeStandard].Charge_Time,
-          chargeSpeed: data.Charge_Standard_Table[chargeStandard].Charge_Speed,
-        };
-        chargeStandardTables.push(chargeStandardTable);
+      let chargeStandardTableUniversal;
+      if (data.Charge_Standard_Table_UK) {
```

Comment:
Is there a reason why we prioritise UK > NL > DE for the standard table ?

Reply:
The Charge_Standard_Table_UK is not allowed to be null according to the documentation. 
However, for some cases it's null in UK and provided in NL and DE.